### ISSUE TITLE
Jodit-react register command

### ISSUE THREAD

**Issue by Tarnn** [jodit/jodit-react#229]
Is there a way to registerCommand? A custom one or to overwrite an existing one like for the ul list, insertUnorderedList.

I saw that the Jodit types has registerCommand but you can't reference it?

**Comment by xdan** [comment]
Yes — registerCommand lives on the editor instance, so grab it via editorRef (or ref):

```jsx
<JoditEditor editorRef={jodit => {
  jodit.registerCommand('insertUnorderedList', (cmd, second, third) => {
    // your override; return false to also run the native behaviour
  });
}} />
```

Overwriting an existing command name replaces the built-in handler. Closing as answered.
